package.json

```diff
@@ -76,7 +76,7 @@
   "devDependencies": {
     "@axe-core/playwright": "^4.7.1",
     "@babel/core": "^7.21.8",
-    "@playwright/test": "^1.34.1",
+    "@playwright/test": "^1.35.1",
```

Comment:
This should be in another PR as I don't think it is required here? It is better to keep PRs focused